src/graph/ShowExecutor.cpp

```diff
@@ -1180,6 +1183,65 @@ std::string ShowExecutor::roleToStr(nebula::cpp2::RoleType type) {
     return role;
 }
 
+std::string ShowExecutor::timestampToStr(int64_t t) {
```

Comment:
maybe this function can move to a common file

Reply:
Good catch
I will do it in nebula  2.0